📝 Update methods section to describe review process (#81)
greenelab/covid19-review: content/60.methods.md

### Article Selection and Evaluation

> The authors solicited relevant articles to be submitted via GitHub [@url:https://Github.com] for review. Articles were classified as _diagnostic_, _therapeutic_, or _other_.

Reviewer: ```suggestion
The authors solicited relevant articles to be submitted via GitHub [@url:https://Github.com] for review. 
Articles were classified as _diagnostic_, _therapeutic_, or _other_.
```

Following a framework often used for assessing medical literature, the review consisted of examining the methods used in the article, the assignment (whether the study was observational or randomized), the assessment, the results, the interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367]. 

#### Diagnostic Papers

##### Methods 

Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered.

> They then described the reference test or "gold standard" if one was utilized. 

Reviewer: ```suggestion
They then described the reference test or "gold standard," if one was utilized. 
```

##### Assignment

Reviewers described how the new and reference tests were assigned and any further details about the study design, for example whether the diagnostic test was biased towards sicker or healthier individuals or very clear-cut positive/negative cases.

##### Assessment

> Reviewers described how the test was performed. For example, if provided, reviewers described the technical details of the assays used, when measurements were taken and by whom for both the standard and reference diagnostic tests.

Reviewer: ```suggestion
Reviewers described how the test was performed. 
For example, if provided, reviewers described the technical details of the assays used, when measurements were taken and by whom for both the standard and reference diagnostic tests.
```

> They then described how individuals are classified as positive or negative and whether there is evidence that the test results are precise or reproducible when repeated more than once.

Reviewer: ```suggestion
They then described how individuals were classified as positive or negative and whether there was evidence that the test results were precise or reproducible when repeated more than once.
```

> Reviewers describe whether there is any missing data, whether some participants only undergo one test, or whether there are individuals with inconclusive results. 

Reviewer: ```suggestion
Reviewers described whether there was any missing data, whether some participants underwent only one test, or whether there were individuals with inconclusive results. 
```

##### Results

> Reviewers report the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV) as well as the confidence bounds around these measures, if provided.

Reviewer: ```suggestion
Reviewers reported the estimated sensitivity, specificity, positive predictive value (PPV), and negative predicted value (NPV), as well as the confidence bounds around these measures, if provided.
```

##### Interpretation

> Reviewers report how well the test rules in or rules out disease based on the population, if there are identified side effects, and patient adherence.

Reviewer: ```suggestion
Reviewers reported how well the test ruled in or rules out disease based on the population, if there were identified side effects, and patient adherence.
```

##### Extrapolation

> Reviewers describe how well this test will extrapolate outside the measured population.

Reviewer: ```suggestion
Reviewers described how well this test will extrapolate outside the measured population.
```

#### Therapeutic Papers

##### Methods 

Reviewers began by describing the study question(s) being investigated by the article.
They then described the study population, the sample size, the prevalence of the disease in the study population, if in human subjects, the countries / regions considered, the demographics of participants, the setting, and any remaining inclusion / exclusion criteria considered. 

##### Assignment

> Reviewers describe how the treatment is assigned, whether it is an interventional or observational study, whether randomization takes place, etc.

Reviewer: ```suggestion
Reviewers described how the treatment is assigned, whether it was an interventional or observational study, whether randomization took place, etc.
```

##### Assessment

###### Outcome Assessment

> Reviewers describe the outcome that is assessed and evaluate whether is it appropriate given the underlying study question. They describe whether there is any missing data, for example whether there are individuals lost to follow up.

Reviewer: ```suggestion
Reviewers described the outcome that was assessed and evaluated whether it was appropriate given the underlying study question. 
They described whether there was any missing data, for example whether there were individuals lost to follow up.
```

> They then describe whether there are any potential sources of bias, for example lack of blinding in a randomized controlled trial.

Reviewer: ```suggestion
They then describe whether there are any potential sources of bias, for example lack of blinding in a randomized controlled trial.
```
```suggestion
They then describe whether there were any potential sources of bias, for example lack of blinding in a randomized controlled trial.
```

###### Statistical Methods Assessment

> Reviewers describe what statistical methods are used for inference and whether the methods are appropriate for the study.

Reviewer: ```suggestion
Reviewers describe which statistical methods were used for inference and whether the methods were appropriate for the study.
```

> They then describe whether adjustments are made for possible confounders.

Reviewer: ```suggestion
They then described whether adjustments were made for possible confounders.
```

##### Results

> Reviewers describe the estimated association between the treatment and outcome.

Reviewer: ```suggestion
Reviewers described the estimated association between the treatment and outcome.
```

> They describe measures of confidence or statistical significance, if provided.

Reviewer: ```suggestion
They described measures of confidence or statistical significance, if provided.
```

##### Interpretation

> Reviewers describe whether a causal claim can be made.

Reviewer: ```suggestion
Reviewers described whether a causal claim could be made.
```

> They describe whether any side effects or interactions with other drugs are identified as well as any subgroup findings.

Reviewer: ```suggestion
They described whether any side effects or interactions with other drugs were identified, as well as any subgroup findings.
```

### Article Selection and Evaluation

> The authors solicited relevant articles to be submitted via GitHub [@url:https://Github.com] for review. 

Reviewer 2: ```suggestion
The authors solicited relevant articles to be submitted via [GitHub](https://github.com/greenelab/covid19-review) for review. 
```

What do you think about linking to this repo instead of the main GitHub site?  I also tend to use standard Markdown links instead of Manubot citations for URLs like this that don't have specific authors or dates associated with them.
